README.md

```diff
@@ -1632,6 +1632,9 @@ arXiv, 2020.
 - <a name="Haessig20fnins"></a>Haessig, G., Milde, M.B., Aceituno, P.V., Oubari, O., Knight, J.C., van Schaik, A., Benosman,  R. B., Indiveri, G.,  
 *[Event-Based Computation for Touch Localization Based on Precise Spike Timing](https://doi.org/10.3389/fnins.2020.00420)*,  
 Front. Neurosci. (2020), 14:420.
+- <a name="Naeini20sensors"></a>Naeini, F. B., Makris, D., Dongming, G., Zweiri, Y.,
```

Comment:
Currently, the authors and title of the publication are on the same line. You need to add two white spaces after the authors to trigger a line break on the github markdown. Could you do this, please? (See the preview to double check).

Reply:
I apologize for the typo. I hope this is fixed in the new commit.